src/components/SRNEntry.vue

```diff
@@ -130,13 +133,30 @@ export default {
         this.validateCount,
         this.isSingleEntryOnly,
         this.redirectID,
-        this.doesUserExist
+        this.isUserValid,
+        this.purpose,
+        this.purposeParams,
+        this.redirectTo
       );
 
       userIsValidated.then((result) => {
-        this.doesUserExist = result.doesUserExist;
+        this.isUserValid = result.isUserValid;
         this.validateCount = result.validateCount;
         this.invalidLoginMessage = result.invalidLoginMessage;
+
+        // either the user is valid or the user has been checked twice
+        if (this.isUserValid || this.validateCount != 1) {
```

Comment:
No, because the loop should be entered when`validateCount == 2` which means the user was authenticated twice. 
The only time it shouldn't be is when `validateCount == 1`, which means the user was authenticated only once.

Reply:
Yes yes it means the same thing, I meant that writing `>` will be more intuitive because, well, if you say it in English, it's like
either the user is valid, or the user has been validated more than once, only then enter the if block

Reply:
Agree with @deepansh96 , fwiw